src/rust/cryptography-x509/src/common.rs

```diff
@@ -234,6 +234,22 @@ pub const PSS_SHA1_HASH_ALG: AlgorithmIdentifier<'_> = AlgorithmIdentifier {
     params: AlgorithmParameters::Sha1(Some(())),
 };
 
+// RSA-PSS ASN.1 hash algorithm definitions specified under the CA/B Forum BRs.
```

Comment:
Is this comment really correct? It's not really defined by CABF, these come from the RFCs, right?

Reply:
I think that's right, although CABF doesn't make reference to the RFCs for their definitions of these. Want me to dig them up regardless?

Reply:
https://www.rfc-editor.org/rfc/rfc8017 looks like the right RFC, although it doesn't explicitly parametrize these the way CABF does.

Reply:
Hmm, really? I just assumed that these were all natural consequences of how it worked, and didn't need to be explicitly specified (same as SHA1)

Reply:
Yeah, CABF specifies the precise set that can be used (and provides their exact DER encodings as well) -- both are on page 97 of the PDF I linked in the top.

Reply:
Ok